client/java-armeria/src/main/java/com/linecorp/centraldogma/client/armeria/AbstractArmeriaCentralDogmaBuilder.java

```diff
@@ -52,10 +56,18 @@
         extends AbstractCentralDogmaBuilder<B> {
 
     private static final int DEFAULT_HEALTH_CHECK_INTERVAL_MILLIS = 15000;
+    private static final ClientOptions DEFAULT_HEALTH_CHECK_CLIENT_OPTIONS =
+            ClientOptions.builder()
+                         .decorator(RetryingClient.newDecorator(
+                                 RetryConfig.builder(RetryRule.onUnprocessed())
+                                            .maxTotalAttempts(3)
```

Comment:
I'm not sure what is a sensible default for retry. 🤔
How about setting `maxTotalAttempts` to 1 so that a client removes unhealthy endpoints quickly?

Reply:
Note: after discussion with @minwoox @ikhoon we decided to try using armeria's default `retryInterval`.

We believe at the time of writing the default health check interval (15s) is a bug.
This is because pre-`centraldogma-0.53.1` the constructor waited only 10s, not allowing for any health check retries on failure.

Reply:
Makes sense.